pkgs/beats.nix

```diff
@@ -0,0 +1,57 @@
+{ lib, fetchFromGitHub, buildGoModule, libpcap, nixosTests, systemd }:
```

Comment:
Not really, upstream versions follow the ELK stack which is not relevant to us anymore as we fixed the ELK version to 7.10.2 to avoid running into licensing issues with Elasticsearch/Kibana. We need newer beats versions and updating the whole ELK stack upstream would be a lot of effort for us.